platform/sun50i/devices.c

```diff
@@ -36,6 +39,15 @@ static struct device msgbox = {
 	.irqdev   = &r_intc,
 };
 
+static struct device pio = {
+	.name     = "pio",
+	.clock    = CCU_GATE(CCU_GATE_PIO) | CCU_RESET(CCU_RESET_PIO),
+	.clockdev = &ccu,
+	.drv      = &sunxi_pio_driver,
+	.drvdata  = 0xFE, /*< Physically implemented PIO ports (1-7). */
```

Comment:
This could be `BITMASK(1, 7)` but I don't know if it's better

Reply:
I will update this; I think that's more descriptive.

Reply:
Done.